test/models/checkCash.ts

```diff
@@ -1,7 +1,9 @@
-import { ValidationError } from 'xrpl-local/common/errors'
-import { verifyCheckCash } from './../../src/models/transactions/checkCash'
 import { assert } from 'chai'
+
+import { ValidationError } from 'xrpl-local/common/errors'
+
 import { verify } from '../../src/models/transactions'
+import { verifyCheckCash } from '../../src/models/transactions/checkCash'
```

Comment:
I'm assuming the linter changed the ordering of all these files? Is that true?

Reply:
Yes - I didn't make that many changes really, I just ran the autolinter and there were a bunch of changes from there.